ITEventsTests/SelectedEventServiceTests.swift

```diff
@@ -0,0 +1,37 @@
+import XCTest
+@testable import ITEvents
+
+class SelectedEventServiceTests: XCTestCase {
+    var selectedEventService: SelectedEventService!
```

Comment:
All your code in tests should also work with a protocol not with implementation:
```swfit
var selectedEventService: ISelectedEventService!
```